core/confirmation.go

```diff
@@ -79,10 +85,10 @@ func (n *OpenBazaarNode) NewOrderConfirmation(contract *pb.RicardianContract, ad
 
 			oc.RatingSignatures = append(oc.RatingSignatures, rs)
 		}
-		oc.PaymentAddress = contract.BuyerOrder.Payment.Address
+		oc.PaymentAddress = order.Payment.Address
 	}
 
-	oc.RequestedAmount = contract.BuyerOrder.Payment.Amount
+	oc.BigRequestedAmount = big.NewInt(int64(order.Payment.Amount)).String()
```

Comment:
I expected the result from `ToV5Order` to have the amount normalized into the `order.Payment.BigAmount` string instead of this int value. Is this a mistake?